### ISSUE TITLE
Allow keyboard selection of history view filters entries

### ISSUE THREAD

**Issue by vincent-legoll** [StackStorm/st2web#438]
in the action combobox popup, when you start to type, it will limit the entries shown in the menu, but you cannot select one (or multiple ones) with the keyboard only, you have to reach the mouse to choose one.

When there's a single one left you should be able to just press enter and it would take that one.

Same for deselection, but this is less problematic, as you already have the mouse in hands to select the combobox, and the active filter is the one at the top of the list.

**Comment by christopherjbaker** [comment]
This has been fixed in the [react rewrite](https://github.com/StackStorm/st2web/pull/420), which should be included with 2.6, though this feature may not be included with the initial release.
